skills/repo-gardener/scripts/config_check.py

```
@@ -379,25 +393,489 @@ def normalize_audit_commands(value: Any, label: str) -> list[list[str]]:
+    options_ended = False
+    while index < len(command):
+        token = command[index]
+        if not options_ended:
+            if token in {"--", "-"}:
+                options_ended = True
+                index += 1
+                continue
+            if (
+                token in {"-S", "--split-string"}
+                or token.startswith("-S")
+                or token.startswith("--split-string=")
```

Ruff (0.16.2)

[warning] Call `startswith` once with a `tuple`

Merge into a single `startswith` call

(PIE810)